napari/layers/points/points.py

```diff
@@ -180,8 +176,8 @@ class Points(Layer):
         that property will be displayed. Multiple properties can be composed using f-string-like
         syntax (e.g., '{property_1}, {float_property:.2f}).
         For example usage, see /napari/examples/add_points_with_text.py.
-    symbol : str
-        Symbol used for all point markers.
+    symbol : str, array
+        Array of symbols for each point.
```

Comment:
this means that it needs to be `array of str` or could be a single string if all symbols are same?

Reply:
You're right, the returned value will always be an array.